include/module.modulemap

```diff
@@ -0,0 +1,67 @@
+module rcheevos {
+    umbrella header "rcheevos.h"
+    
+    module rapi {
+        module rc_api_editor {
+            header "rc_api_editor.h"
+            export *
+        }
+        
+        module rc_api_info {
+            header "rc_api_info.h"
+            export *
+        }
+        
+        module rc_api_request {
+            header "rc_api_request.h"
+            export *
+        }
+        
+        module rc_api_runtime {
+            header "rc_api_runtime.h"
+            export *
+        }
+        
+        module rc_api_user {
+            header "rc_api_user.h"
+            export *
+        }
+
+        export *
+    }
+    
+    module rc_client {
+        header "rc_client.h"
+        export *
+    }
+    
+    module rc_consoles {
+        header "rc_consoles.h"
+        export *
+    }
+    
+    module rc_error {
+        header "rc_error.h"
+        export *
+    }
+    
+    module rc_runtime {
+        header "rc_runtime.h"
+        header "rc_runtime_types.h"
+        export *
+    }
```

Comment:
`rc_client` is definitely a runtime file. `rc_consoles` and `rc_error` are common files shared by the other modules. Their supporting code is with the other runtime files. I'm not sure if they warrant a "common" module, or should just be glopped in with the runtime files.

Reply:
The reason I put `rc_runtime.h` and `rc_runtime_types.h` together is because I didn't know if the strange double-declaration of types would cause issues.

I'll just split them up as it doesn't seem to actually cause issues.